app/parse.py

```diff
@@ -8,5 +13,28 @@ class Course:
     duration: str
 
 
+def get_course(course: Tag) -> Course:
+    return Course(
+        name=course.select_one(".ProfessionCard_title__Zq5ZY > h3").text,
+        short_description=(
+            course.select_one(".typography_landingTextMain__Rc8BD.mb-32").text
+        ),
+        duration=(
+            course.select(
+                ".ProfessionCardTags_regularTag__yTc6K"
+                ".ProfessionCardTags_regularTag__yTc6K"
```

Comment:
The CSS selector '.ProfessionCardTags_regularTag__yTc6K' is repeated, which is likely a mistake. This could lead to incorrect extraction of the course duration. Ensure that the selector accurately targets the intended element.